internal/cmd/schema-registry/command_cluster_delete.go

```diff
@@ -63,6 +63,12 @@ func (c *command) clusterDelete(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
+	ctx := c.Config.Context()
+	ctx.SchemaRegistryClusters[environmentId] = nil
+	if err := ctx.Save(); err != nil {
```

Comment:
If the cluster is deleted but the configuration file is not saved, we should still print the message below. Can we ignore this error?

Reply:
good point we should still print the `deleted` msg. But idk about ignoring the `save config` error. Is there any other places that we're ignoring it?

Reply:
Should be fine! Searching through the repo I see 4-5 other places where we ignore the error!